src/test/scenarios/datafactory/configuration/readme.cli.md

```diff
@@ -31,6 +30,8 @@ cli:
     - name: /IntegrationRuntimes/post/IntegrationRuntimes_Upgrade
     - name: /IntegrationRuntimes/patch/IntegrationRuntimes_Update
     - name: /Triggers/put/Triggers_Create
+    - name: /Triggers/put/Triggers_Create
+    - name: /Triggers/put/Triggers_Update
     - name: /Triggers/put/Triggers_Update
```

Comment:
Is this duped with the one you added?

Reply:
this should be the generic create/update problem in test, I'll check it.

Reply:
Okay, then you should match the operation CreateOrUpdate#Update